.github/workflows/pip.yml

```diff
@@ -4,11 +4,13 @@
 
 name: Build PyPI package
 
-on:
-  push:
-    branches: [ main ]
-  release:
-    types: [ created ]
+#on:
```

Comment:
Is this commented out code needed?

Reply:
It is (unfortunately) the only way I've found to test GitHub Actions (ie, by temporarily changing the trigger action to allow it to actually run before checking it in). I should have done this before requesting approval, sorry.